src/config/PluginConfigManager.ts

```diff
@@ -101,7 +102,7 @@ export class PluginConfigManager<TPluginType extends BasePluginType> {
 
     // Passed category id -> passed message's channel's category id
     const categoryId =
-      matchParams.categoryId || (message && message.channel && (message.channel as GuildChannel).parentID);
+      matchParams.categoryId || (message && message.channel && (message.channel as GuildChannel).parentId);
```

Comment:
There might be prettier solutions, but this is sufficient compared to what was there beforehand

```suggestion
      matchParams.categoryId || (message?.channel && (message.channel as GuildChannel).parentId);
```